docs/models/google.md

```diff
@@ -201,6 +201,71 @@ agent = Agent(model)
 
 `GoogleModel` supports multi-modal input, including documents, images, audio, and video. See the [input documentation](../input.md) for details and examples.
 
+## Enhanced JSON Schema Support
```

Comment:
Let's remove this entire docs section; we transform the schemas exactly so the user doesn't have to know about these details.